module/User/src/Entity/User.php

```diff
@@ -1,60 +1,81 @@
 <?php
-
 namespace User\Entity;
 
 use DateTime;
 use ZfcRbac\Identity\IdentityInterface;
 
 class User implements IdentityInterface
 {
+
     const ROLE_GUEST = 'guest';
+
     const ROLE_REGISTERED = 'registered';
-    
+
     /**
+     *
      * @var integer
      */
     protected $id;
+
     /**
+     *
      * @var string
      */
     protected $person;
+
     /**
+     *
      * @var string
      */
     protected $email;
+
     /**
+     *
      * @var string
      */
     protected $password;
+
     /**
+     *
      * @var string
      */
     protected $roleId;
+
     /**
+     *
      * @var boolean
      */
     protected $emailConfirm = false;
+
     /**
+     *
      * @var string
      */
     protected $confirmKey;
+
     /**
+     *
      * @var DateTime
      */
     protected $createdAt;
+
     /**
+     *
      * @var string|null
      */
     protected $restoreKey;
+
     /**
+     *
      * @var DateTime|null
      */
     protected $restoreKeyExpire;
 
     /**
+     *
      * @return integer|null
      */
-    public function getId(): ?int
+    public function getId(): int
```

Comment:
How to checking that an entity is new or exists in database without NULL? Why you deleted `?`?